Corona-Warn-App/src/main/java/de/rki/coronawarnapp/nearby/modules/tekhistory/DefaultTEKHistoryProvider.kt

```diff
@@ -18,25 +18,25 @@ class DefaultTEKHistoryProvider @Inject constructor(
     private val client: ExposureNotificationClient
 ) : TEKHistoryProvider {
 
-    override suspend fun isTEKAccessPermissionGranted(): Boolean {
-        return try {
-            getTEKHistory()
-            true
-        } catch (e: ApiException) {
-            if (e.statusCode == ExposureNotificationStatusCodes.RESOLUTION_REQUIRED) {
-                false
-            } else {
-                throw e
-            }
+    override suspend fun isTEKAccessPermissionGranted(): Boolean = try {
+        getTEKHistory()
```

Comment:
Doesn't this prompt for the native popup where the user has to grant access?!

Reply:
Correct, it was a leftover, my initial API design added it, then due to the concerns you mentioned it was never used. Removed it.